LIPS/lip-23.md

```diff
@@ -80,6 +80,7 @@ interface SecondOpinionOracle {
 	function getReport(uint256 refSlot) external view returns ( 
 		bool success, 
 		uint256 clBalanceGwei, 
+		uint256 withdrawalVaultBalance, 
 		uint256 totalDepositedValidators, 
 		uint256 totalExitedValidators 
 	); 
```

Comment:
The note below might be misleading, please state what parameters are mandatory for the report